src/utils/features.js

```diff
@@ -19,3 +19,10 @@ export const getProviderIntegrationConfig = providerName =>
   configInterface.providers.options &&
   configInterface.providers.options[providerName.toUpperCase()]) || {}
 
+export const isThirdPartyIntegrationEnabled = thirdPartyName =>
```

Comment:
why didn't you use optional chaining operator? 
```
config.thirdparty?.[thirdPrtyName]?.enabled
```
https://github.com/tc39/proposal-optional-chaining

Reply:
Because VSCode doesn't like it 😉 Will do though

Reply:
Not necessary if you don't like it, It's just a personal preference 😄